syntax = "proto2";

package Onion5;

import "google/protobuf/timestamp.proto";

message Node {
  required string github_id = 1;
  required string ip_addr = 2;
  required google.protobuf.Timestamp last_check = 3;
}

message NodeList {
  repeated Node nodes = 1;
}
